sickbeard/processTV.py

```diff
@@ -665,3 +670,25 @@ def subtitles_enabled(video):
     else:
         logger.log(u'Empty indexer ID for: {}'.format(video), logger.WARNING)
         return
+ 
+def has_embedded_subtitles(video):
```

Comment:
@ratoaq2 with this we will import subliminal that will import enzyme right? 

https://github.com/Diaoul/subliminal/blob/develop/subliminal/refiners/metadata.py#L6

so why not do as I did an import enzyme directly?